cpp/include/Ice/Communicator.h

```diff
@@ -19,7 +19,7 @@
 #include <Ice/Incoming.h>
 #include <Ice/IncomingAsync.h>
 #include <Ice/FactoryTableInit.h>
-#include <Ice/Optional.h>
+#include <optional>
```

Comment:
The std imports should be sorted below the Ice imports.

Reply:
These are all "local generated code" files that need to be refactored.